### PR TITLE
Fix doc

### PR DESCRIPTION
**Pull request by uwedeportivo**
make doc @ 3.14 work

### CODE REVIEW

File: doc/index.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ We actively maintain the two most recent monthly releases of Sourcegraph.
 Upgrades should happen across consecutive minor versions of Sourcegraph. For example, if you are
 running Sourcegraph 3.1 and want to upgrade to 3.3, you should upgrade to 3.2 and then 3.3.
 
-> The Docker server image tags follow SemVer semantics, so version 3.13.2 can be found at `sourcegraph/server:3.13.2`. You can see the full list of tags on our [Docker Hub page](https://hub.docker.com/r/sourcegraph/server/tags).
+> The Docker server image tags follow SemVer semantics, so version 3.13.2 can be found at `sourcegraph/server:3.14.3`. You can see the full list of tags on our [Docker Hub page](https://hub.docker.com/r/sourcegraph/server/tags).
```
Comment (unknwon) [inline doc/index.md:52]: no no... haha

Comment (uwedeportivo) [reply doc/index.md:52]: too late :-)

**Comment by uwedeportivo** [comment]
we have to stay authentic

File: doc/index.md (hunk 1 above)
Comment (unknwon) [reply doc/index.md:52]: When can we end this game 😭

**Comment by uwedeportivo** [comment]
soon

**Comment by uwedeportivo** [comment]
i'm making a PR that fixes the script

**Pull request merged**